packages/nextjs/app/admin/page.tsx

```diff
@@ -5,28 +5,41 @@ import { BatchActionModal } from "./_components/BatchActionModal";
 import { GrantReview } from "./_components/GrantReview";
 import useSWR from "swr";
 import { parseEther } from "viem";
+import { useAccount } from "wagmi";
 import { useScaffoldContractWrite } from "~~/hooks/scaffold-eth";
 import { GrantDataWithBuilder } from "~~/services/database/schema";
 import { PROPOSAL_STATUS } from "~~/utils/grants";
 import { notification } from "~~/utils/scaffold-eth";
 
+const fetcherWithHeader = (url: string, address: string) =>
+  fetch(url, {
+    headers: {
+      Address: address,
+    },
+  }).then(res => res.json());
```

Comment:
To make https://github.com/BuidlGuidl/grants.buidlguidl.com/pull/96/files#r1539245051 work, we need to handle `res.ok` case example : 

```ts
const fetcherWithHeader = async (url: string, address: string) => {
  const res = await fetch(url, {
    method: "GET",
    headers: {
      Address: address,
    },
  });
  const data = await res.json();
  // check if server send 401 or any other error
  if (!res.ok) {
    throw new Error(data.error || `Error getting data`);
  }
  return data;
};
```